### PR TITLE
docs - update bigquery docs

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Updates BigQuery connection docs to include new sync settings
- Adds a note about connecting to external data sources: google drive
- Updates some docs on caching, including the paid database-wide TTL setting
- Does some formatting and other cleanup

Questions:

- For the comma-separated list of datasets to include/exclude: Is the `*` wildcard the only special char people can use to select datasets? Are there other chars people can use? Can they use full regexes?
- I updated the copy in driver/common.clj. Is that the right copy? (Before this change, we had duplicate copy for both the sync and scan and refingerprint options.)
- Any key info missing from for the BigQuery docs?

For now, I'm okay with repeating some copy for database-specific pages (e.g., sync settings that apply to all databases), as the set of settings differs between databases. But we could look into autogenerating docs for these connection pages (like pulling these descriptions out of defs and putting them into markdown files/json/yaml or some other format - or pull from the defs themselves).

### CODE REVIEW

File: docs/administration-guide/databases/bigquery.md [hunk 1]
```diff
@@ -34,52 +34,98 @@ Once you have created and downloaded your service account JSON file for your Big
...
-#### Use the Java Virtual Machine (JVM) timezone
+### Datasets
+
+Here you can specify which datasets you want to sync and scan. Options are:
+
+- All
+- Only these...
+- All except...
+
+For the Only these and All except options, you can input a comma-separated list of values to tell Metabase which datasets you want to include (or exclude). For example,:
```
Comment (frannietrempe) [inline docs/administration-guide/databases/bigquery.md:61]:
```suggestion
For the Only these and All except options, you can input a comma-separated list of values to tell Metabase which datasets you want to include (or exclude). For example:
```

**Review by frannietrempe — APPROVED** [review]

**Comment by jeff303** [comment]
> * For the comma-separated list of datasets to include/exclude: Is the `*` wildcard the only special char people can use to select datasets? Are there other chars people can use? Can they use full regexes?

Only the `*` wildcard character is supported.  No generic regex syntax.

> * I updated the copy in driver/common.clj. Is that the right copy? (Before this change, we had duplicate copy for both the sync and scan and refingerprint options.)

Yes, the change looks right to me.

> * Any key info missing from for the BigQuery docs?

I had a look at the changes to the BigQuery docs, and everything there looks good to me.  It might be useful to have a concrete example of the "exclusion" scenario for dataset filtering (i.e. if you have concrete datasets `foo`, and `bar`, then a pattern of `b*` will result in only tables from `foo` being included, or something along those lines).  But that's your call.

**Comment by flamber** [comment]
@jeff303 Does BigQuery now support timezones? https://www.metabase.com/docs/latest/troubleshooting-guide/timezones.html

**Comment by jeff303** [comment]
> @jeff303 Does BigQuery now support timezones? https://www.metabase.com/docs/latest/troubleshooting-guide/timezones.html

Unfortunately, [still no](https://github.com/metabase/metabase/blob/master/modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj#L285-L288).

**Pull request merged**